unittests/FEXLinuxTests/tests/signal/invalid_int1.cpp

```diff
@@ -9,7 +13,7 @@ __attribute__((naked, nocf_check))
 static void InvalidINT() {
   __asm volatile(R"(
   .byte 0xF1; # int1
-  ret; # Just incase it gets past the int
```

Comment:
This line should be removed from the other `invalid_` tests too, since this is now the expected path of execution.